website/docs/docs/cloud/manage-access/set-up-sso-okta.md

```diff
@@ -156,14 +156,7 @@ the integration between Okta and dbt Cloud.
 
 ## Configuration in dbt Cloud
 
-To complete setup, follow the steps below in dbt Cloud.
-
-### Enable Okta Native Auth (beta)
-
-There are two ways to enable Okta depending on how you access dbt Cloud:
-
-- If you access dbt Cloud using an [Access URL](/docs/cloud/about-cloud/regions-ip-addresses), such as `cloud.getdbt.com`, contact your account manager to gain access to the Okta configuration user interface.
-* If you access dbt Cloud using virtual private cloud (VPC), enable the `native_okta` feature flag in the dbt Cloud admin backend.
+To complete setup, follow the steps below in dbt Cloud. If you access dbt Cloud using virtual private cloud (VPC), enable the `native_okta` feature flag in the dbt Cloud admin backend.
```

Comment:
looking at the preview, sugg keeping the existing IA (ie, both the H2 and H3 sections) since the `native_okta` fflag is still active for some people. unsure about the `(beta)` in the H3 tho, whether to keep or remove.    

```suggestion
## Configuration in dbt Cloud

To complete setup, follow the steps below in dbt Cloud. 

### Enable Okta native auth (beta)

If you access dbt Cloud using virtual private cloud (VPC), enable the `native_okta` feature flag in the dbt Cloud admin backend.

```